src/rules/double-slash-comment-inline/__tests__/index.js

```diff
@@ -0,0 +1,160 @@
+import testRule from "stylelint-test-rule-tape"
+import rule, { ruleName, messages } from ".."
+
+// -------------------------------------------------------------------------
+// "always"
+// -------------------------------------------------------------------------
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  syntax: "scss",
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "a {} // Inline comment, outside a ruleset.",
+    description: "Inline comment, outside a ruleset.",
+  }, {
+    code: `a { // Inline comment, after {.
+      width: 10px;
+    }`,
+    description: "Inline comment, after {.",
+  }, {
+    code: `
+      a, // Inline comment between selectors.
+      b {
+        width: 10px;
+      }
+    `,
+    description: "Inline comment between selectors.",
+  }, {
+    code: `a {
+      /* Non-inline CSS comment, ignored */
+      width: 10px;
+    }`,
+    description: "Non-inline CSS comment, ignored.",
+  } ],
+
+  reject: [ {
+    code: `
+      a { }
+      // Non-inline comment (after a ruleset)
+    `,
+    description: "Non-inline comment (after a ruleset)",
+    message: messages.expected,
+    line: 3,
+    column: 7,
+  }, {
+    code: `
+      a,
+      // Non-inline comment between selectors
+      b { }
+    `,
+    description: "Non-inline comment between selectors.",
+    message: messages.expected,
+    line: 3,
+    column: 7,
+  }, {
+    code: `
+      a {
+        // Non-inline comment (before a decl)
+        width: 10px;
+      }
+    `,
+    description: "Non-inline comment (before a decl)",
+    message: messages.expected,
+    line: 3,
+    column: 9,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: [ "always", { ignore: ["stylelint-commands"] } ],
+  syntax: "scss",
+  skipBasicChecks: true,
+
+  accept: [{
+    code: `
+      a {
+        color: pink;
+        // stylelint-disable something 
```

Comment:
Same thing here, I think that stylelint only supports /* comments for stylelint-disable.